packages/aragon-cli/package.json

```diff
@@ -90,7 +90,7 @@
     "truffle-flattener": "^1.2.9",
     "web3": "^1.2.4",
     "which": "^1.3.1",
-    "yargs": "^13.2.4"
+    "yargs": "yargs/yargs"
```

Comment:
Is this correct?

Reply:
Yes, that's to install the current Github master since the version with the feature is not released yet. Maybe it would be better to lock it to a specific commit